### PR TITLE
Make admin main page

### PR DESCRIPTION
**Pull request by AlexSukov**

### CODE REVIEW

**Comment by AlexSukov** [comment]
Vanya, look pls at the add post page. We have an issue if we don't put user_id in the User field.

**Comment by AlexSukov** [comment]
And tell pls, how to make a link_to devise links properly.

**Comment by Magons** [comment]
@vakyl9r and @AlexSukov, use `@` to select recipient name who you address your comment

File: app/models/post.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Post < ApplicationRecord
   has_many :comments
   belongs_to :user
+
```
Comment (Magons) [inline app/models/post.rb]: extra line

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -11,4 +11,5 @@
  * It is generally better to create a new file per style scope.
  *
  */
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
+
```
Comment (Magons) [inline app/assets/stylesheets/application.scss]: extra line

File: app/views/posts/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  %tbody
+    - @posts.each do |post|
+      %tr
+        %td= post.title
+        %td= post.body
+        %td= post.user.email
+        %td= link_to 'Показать', post
+        %td= link_to 'Выход', edit_post_path(post)
+        %td= link_to 'Удалить', post, method: :delete, data: { confirm: 'Вы уверены?' }
+
```
Comment (Magons) [inline app/views/posts/index.html.haml]: extra line

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 class PostsController < ApplicationController
   before_action :set_post, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!, only: [:new, :edit, :update, :destroy]
```
Comment (Magons) [inline app/controllers/posts_controller.rb]: move this up

File: app/views/posts/index.html.haml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+%p#notice= notice
+
+%h1 БЛОГ
```
Comment (Magons) [inline app/views/posts/index.html.haml]: use ruby code
`%h1= 'БЛОГ'`

File: app/views/posts/index.html.haml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+= link_to 'Новый пост', new_post_path
+%table.table
+  %thead
+    %tr
+      %th Название
```
Comment (Magons) [inline app/views/posts/index.html.haml]: the same

File: config/routes.rb [hunk 7]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root to: 'posts#index', as: 'post'
```
Comment (Magons) [inline config/routes.rb]: do we need remove `as: 'post'`?

**Comment by Magons** [comment]
@AlexSukov make rebase please

**Pull request merged**